_posts/2018-10-12-29.markdown

```diff
@@ -9,29 +9,10 @@ slug: 'speckandtech-retreat'
 ## The retreat
 We gathered over 50 of our community members to spend some quality time together! Feel like taking a look at all the details? [Check the mini-site of the event](https://speckand.tech/retreat).
 
-## The aftermovie
```

Comment:
@franzbona perchè vuoi rimuovere tutta sta roba?

Reply:
L'idea era di tenere il Retreat come un evento più "intimo" ed esclusivo, e quindi non buttare fuori i filmati dell'evento (infatti quest'anno non abbiamo nemmeno ripreso o fatto la live), a parte l'aftermovie, oppure le slides.

Reply:
Provo a vedere le modifiche in locale, ma ok 👍 capito

Reply:
Non c'è alcuna modifica sostanziale, solo rimozioni di embed 😄 safe!